### ISSUE TITLE
Dashboard Temperature to Fahrenheit and change Picture Icon with correct Birthdate

### ISSUE THREAD

**Issue by rockman55** [ivadim/fruitnanny#3]
I would like to know where I can substitute the picture on the upper left corner of the Dashboard and the date of birth for that person. Also I would like to display the temperature in Fahrenheit, not Celsius.
Thanks,
Danny

**Comment by ivadim** [comment]
Hi @rockman55 , now you can configure application 😄 
Pull latest changes and modify `fruitnanny_config.js` .

Params:
* `baby_name` - baby's name to display in UI
* `baby_birthday` - baby's birthday
* `temp_unit` - temperature to display in Celsius (`C`) or Fahrenheit(`F`)

(in your case put F in temp_unit)

To update baby's picture you need to replace file `public\project\img\baby.png`.

**Comment by rockman55** [comment]
Hello Dmitry

I made the changes you mentioned, and I started with a clean install.

Now the baby picture has changed, the birthdate is correct, and the temperature displays correctly.

However, I am not getting any video or sound. Have I made some mistake or something else?

Can these changes be made from a previous image I have that works just by changing the fruitnanny_config.js file or what would be the correct procedure with the fewest changes?

 

Thanks,

Danny
